src/components/dialogs/network-modifications/equipment-deletion/equipment-deletion-dialog.js

```diff
@@ -7,7 +7,15 @@
 
 import { yupResolver } from '@hookform/resolvers/yup';
 import yup from 'components/utils/yup-config';
-import { TYPE, EQUIPMENT_ID } from '../../../utils/field-constants';
+import {
+    TYPE,
+    EQUIPMENT_ID,
+    SHUNT_COMPENSATOR_SIDE_1,
+    SHUNT_COMPENSATOR_SIDE_2,
+    HVDC_WITH_LCC,
```

Comment:
I don't think hvdc_with_lcc info is usefull ...

See remarks in following PRs: 
https://github.com/gridsuite/network-modification-server/pull/272#discussion_r1239823115
https://github.com/gridsuite/network-map-server/pull/122#discussion_r1239777488

Reply:
yes/ maybe I can just use mcsOnSideX

Reply:
done